pages/doc/wavefront_spring_boot3_tutorial.md

```diff
@@ -10,6 +10,8 @@ In this tutorial, you use Wavefront for Spring Boot that uses Spring Boot 3 with
 
 {% include tip.html content="Want to start your project from scratch using [https://start.spring.io/](https://start.spring.io/)? Follow the [Observability with Spring](https://spring.io/guides/gs/tanzu-observability/) tutorial." %}
 
+See also the VMware Tanzu solutions workbook [Instrumenting TAS OpenTelemetry for Spring Boot Application](https://docs.vmware.com/en/VMware-Tanzu-Reference-Architecture/services/tanzu-solutions-workbooks/solution-workbooks-TAS-OpenTelemetry-SpringBoot-TO.html).
```

Comment:
Q: Same question as above.

If you are using Tanzu Application Service (TAS) and if you are an existing Aria Operations for Applications customer, you can use the VMware Tanzu solutions workbook. For more information, see [Instrumenting TAS OpenTelemetry for Spring Boot Application](https://docs.vmware.com/en/VMware-Tanzu-Reference-Architecture/services/tanzu-solutions-workbooks/solution-workbooks-TAS-OpenTelemetry-SpringBoot-TO.html).

Reply:
Absolutely agree, adding the "if" clause clarifying that this info is for TAS customers. Thank you!